src/OSPSuite.Core/Chart/ChartFonts.cs

```diff
@@ -25,61 +24,37 @@ public ChartFonts()
       public int AxisSize
       {
          get => _axisSize;
-         set
-         {
-            _axisSize = value;
-            OnPropertyChanged(() => AxisSize);
-         }
+         set => SetProperty(ref _axisSize, value, () => AxisSize);
```

Comment:
this should already prevent some of the events that we saw in case of update because we will set the same property all the time. That being said, let's do it right